### ISSUE TITLE
Best way to set headers?

### ISSUE THREAD

**Issue by CezarLuiz0** [tulios/mappersmith#20]
There is a best way to set `headers` on pass params?

Example:

```
Client.Photo.save({
  category: 'family',
  body: {year: 2015, tags: ['party', 'family']},
  headers: {
      'Authorization': 'token MYTOKEN'
  }
})
```

Thanks!

**Comment by tulios** [comment]
Hi @CezarLuiz0, you need the authorization header only in this call or for all of them? For either option you can use the url matching to configure the headers. Take a look at https://github.com/tulios/mappersmith#global-configurations-and-url-matching

**Comment by CezarLuiz0** [comment]
Hi @tulios, thanks for the quick answer.

I saw the url matching, but I think the docs needs more examples.

Not all my calls need the header, but the vast majority need to set other headers in addition to these.

Thanks!

**Comment by tulios** [comment]
The documentation is covering all the features but I agree with you that it lacks examples, thank you for the feedback. With the URL matching, you can have different headers per group of calls and you don't need to change your main code later, you can just include or remove headers from the manifest. However, I understand the wish of having a more direct way to define the headers.

I will create an issue to implement this and link here.

**Comment by tulios** [comment]
hey @CezarLuiz0, I will release a new version with the changes this week at most, I'm just checking for other needs of headers like the response headers.

**Comment by CezarLuiz0** [comment]
Ok, thanks!

**Comment by tulios** [comment]
@CezarLuiz0 I've published version 0.12.0 with the new header support.
